python/lsst/rubintv/production/uploaders.py

```diff
@@ -271,10 +271,10 @@ def __init__(self):
         remoteRequired = getSite() in ["summit", "tucson", "base"]
         if remoteRequired and not self.hasRemote:
             raise RuntimeError("Failed to create remote S3 uploader")
-        elif remoteRequired and self.hasRemote:
-            remoteOk = self.remoteUploader.checkAccess()
-            if not remoteOk:
-                raise RuntimeError("Failed to connect to remote S3 bucket")
+        # elif remoteRequired and self.hasRemote:
+        #     remoteOk = self.remoteUploader.checkAccess()
+        #     if not remoteOk:
+        #         raise RuntimeError("Failed to connect to remote S3 bucket")
```

Comment:
Please rebase away this commit, even if you've already undone it.  It makes the git logs look worrying.